build.sbt

```diff
@@ -91,9 +91,10 @@ lazy val discourse = project.settings(
     "io.circe"                   %% "circe-core"           % circeVersion,
     "io.circe"                   %% "circe-generic-extras" % circeVersion,
     "io.circe"                   %% "circe-parser"         % circeVersion,
+    "com.typesafe.akka"          %% "akka-http"            % akkaHttpVersion,
```

Comment:
Why did you add this dep?

Reply:
Because I had to force `akka-http-circe` to use this version of akka-http. Otherwise he get conflicting akka versions.